tracer/test/test-applications/regression/AspNetCoreSmokeTest/Program.cs

```diff
@@ -15,9 +15,9 @@ public class Program
         public static volatile int ExitCode = 0;
         public static async Task<int> Main(string[] args)
         {
-            if (!IsProfilerAttached())
+            if (!IsProfilerAttached() && Environment.GetEnvironmentVariable("PROFILER_IS_NOT_REQUIRED") != "True")
             {
-                Console.WriteLine("Error: Profiler is required and is not loaded.");
+                Console.WriteLine($"Error: Profiler is required and is not loaded. PROFILER_IS_NOT_REQUIRED={Environment.GetEnvironmentVariable("PROFILER_IS_NOT_REQUIRED")}.");
```

Comment:
more nit-picking, but 

```suggestion
            var profilerNotRequired = Environment.GetEnvironmentVariable("PROFILER_IS_NOT_REQUIRED");

            if (!IsProfilerAttached() &&  profilerNotRequired != "True")
            {
                Console.WriteLine($"Error: Profiler is required and is not loaded. PROFILER_IS_NOT_REQUIRED={profilerNotRequired}.");
```

Reply:
I almost wrote it like this originally and was too lazy, thanks for keeping me honest 😅